mptt/managers.py

```diff
@@ -616,6 +616,7 @@ def move_node(self, node, target, position='last-child'):
         move the node yourself by setting node.parent.
         """
         self._move_node(node, target, position=position)
+        node.save()
```

Comment:
I'm unsure why this is here. Doesn't `_move_node()` handle writing the changes to the db?